backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/dto/VoteIsOpenDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.cgi.retrospecto.backend.poker.controller.dto;
+
+public class VoteIsOpenDTO {
```

Comment:
Same comment as in CreateRoomDTO and CreateStoryDTO.

Reply:
It is like this because the lack of my JS knowledge. So in frontend I could copy-paste working requests with some modifications. Later I will try to modify the frontend and simplify so these DTOs will be gone.